lua/entities/ent_jack_aidfuel_diesel/init.lua

```diff
@@ -0,0 +1,25 @@
+AddCSLuaFile( "cl_init.lua" )
+AddCSLuaFile( "shared.lua" )
+include( "shared.lua" )
+
+function ENT:Initialize()
+    self:SetModel( "models/props_lab/harddrive02.mdl" )
+    self:SetColor( Color( 31, 31, 31 ) )
```

Comment:
nit: Prefer using a local `Color` instance or `SetColor4Part`

Reply:
It's only called on entity creation, it wont affect it much at all

Reply:
That's why it's a `nit` :)